spec/metrics/scoping_spec.rb

```diff
@@ -42,14 +42,14 @@
 
     it "works with several defaults" do
       expect(dappy.where(user_id: 1).to_sql)
-        .to eq "select * from \"dummy\" group by time(1h) where (user_id=1) limit 100"
+        .to eq "select * from \"dummy\" where (user_id=1) group by time(1h) limit 100"
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver.